src/lib/components/checkbox/checkbox.module.scss

```diff
@@ -6,6 +6,10 @@
 //   flex-direction: column;
 // }
 
+.customCheckbox {
```

Comment:
AH! YES! I had this same thought about the targeting with the adjacent CSS thingo when we talked and then thought about how both would probably be wrapped in wrappers and then it got complicated in my head and I unintentionally moved on and forgot about it :P. I will look into fixing this up now! (Also side note, I have [this link](https://code.tutsplus.com/tutorials/the-30-css-selectors-you-must-memorize--net-16048) bookmarked which you sent me give or take... 10 years ago! Haha. Probably outdated but still a good little cheat sheet for remembering what is possible)

Reply:
👍🏼 // 😀 I remember that link, still very useful

Reply:
I've just made some small changes to move the checkbox up when it comes directly after the t&cs... I did it with a negative margin which I think is acceptable in this instance (normally I would avoid it but because of CSS modules - I think it will work best). Let me know if you wanna speak about my thinking ;) @mischacolley

Reply:
Sounds good @lulen11 👍🏼 Can you just make sure you've tested against https://github.com/aimementoring/aime-portal/pull/376 so you know it's working in context ... also wouldn't be a bad idea to test how it works on smaller screens

Reply:
updated styles and it's looking good now on all sizes :)